application/src/main/resources/schedule.yml

```diff
@@ -0,0 +1,82 @@
+#
+# Copyright 2020 Telefonaktiebolaget LM Ericsson
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+##
+## The schedules can be defined using exact keyspace/table names.
+## It's also possible to define the names using quoted regular expressions.
+##
+## When the repair scheduler configures repairs it adheres to the following order:
+## | Keyspace regex? | Table regex? |
+## |-----------------|--------------|
+## | No              | No           |
+## | No              | Yes          |
+## | Yes             | No           |
+## | Yes             | Yes          |
+##
+## If multiple regular expressions matches a single table there is no preference defined.
```

Comment:
Perhaps something along the lines of:
Multiple regular expressions matching a single table should be avoided as it will be difficult to know which value will win out.

Reply:
Done